STEERING-COMMITTEE.md

```diff
@@ -7,51 +7,55 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) defines the governance for Knative
+in pursuit of the goal of an open, vibrant project with a productive
+community, a healthy ecosystem of vendors, and most of all satisfied
+customers running their workloads on Knative clusters.
```

Comment:
technically there are no customers of knative, there are users. There are customers of managed knative clusters and products based on knative. In general one cannot be a customer of an open source project since the community does not sell anything.